### PR TITLE
fix: the `capsule debugger` command was broken

### PR DESCRIPTION
**Pull request by maq128**
the `capsule debugger` command was broken, as the command line arguments mismatched with the upstream ckb-debugger.

### CODE REVIEW

**Review by TheWaWaR — APPROVED** [review]

**Comment by TheWaWaR** [comment]
Please sign your commits.

**Comment by TheWaWaR** [comment]
You should reset the unsigned commit and then sign and commit the changes:
```
git reset 2b46cfa
git add .
git commit -m "fix: the capsule debugger command arguments was broken"
git push -f
```

**Comment by maq128** [comment]
> You should reset the unsigned commit and then sign and commit the changes:
> 
> ```
> git reset 2b46cfa
> git add .
> git commit -m "fix: the capsule debugger command arguments was broken"
> git push -f
> ```

done and thank you. :)

File: src/debugger.rs [hunk 1]
```diff
@@ -64,6 +65,7 @@ pub fn start_debugger<P: AsRef<Path>>(
     .host_network(true)
     .name(DEBUG_SERVER_NAME.to_string())
     .daemon(tty)
+    .tty(!tty)
```
Comment (TheWaWaR) [inline src/debugger.rs]: tty default is `false`, this line is not necessary.

File: src/debugger.rs [hunk 2]
```diff
@@ -98,8 +100,10 @@ pub fn start_debugger<P: AsRef<Path>>(
         // }
 
         docker_cmd.run(cmd, signal)?;
+
+        return DockerCommand::stop(DEBUG_SERVER_NAME);
     }
-    DockerCommand::stop(DEBUG_SERVER_NAME)
+    Ok(())
```
Comment (TheWaWaR) [inline src/debugger.rs:105]: Why this change? Will this command not working when run with `tty=false`?

File: src/debugger.rs (hunk 1 above)
Comment (maq128) [reply src/debugger.rs]: well, you're right, it is not necessary. :)

File: src/debugger.rs (hunk 2 above)
Comment (maq128) [reply src/debugger.rs:105]: If I understand correctly, `tty=false` comes from `--only-server`, which means ckb-debugger should run in `full` mode (rather than `gdb` mode), in this case, `.run()` will keep waiting for ckb-debugger to finish, after that, `DockerCommand::stop(DEBUG_SERVER_NAME)` is unnecessary, or it will result in error "No such container: capsule-debugger-server".

**Comment by TheWaWaR** [comment]
Your commits still not signed, you may read the doc for more details https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

**Comment by maq128** [comment]
> Your commits still not signed, you may read the doc for more details https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

Sorry, forgot, again.
